en/docs/setup/prerequisites.md

```diff
@@ -18,3 +18,4 @@
      - Add Jetstack
     
          `helm repo add jetstack https://charts.jetstack.io`
+
```

Comment:
Shall we add the required helm version here?

Reply:
Lets add this in separate PR